sdk/cosmos/azure-cosmos-spark_3_2-12/src/test/scala/com/azure/cosmos/spark/utils/CosmosPatchTestHelper.scala

```diff
@@ -219,7 +271,8 @@ object CosmosPatchTestHelper {
   *
   * @return the partition key path without
   */
-  // TODO: Reexamine the logic here when hierarchical partitioning being supported
+  //  TODO: Reexamine the logic here when hierarchical partitioning being supported
+  // for hierarchical partitioning this is not used/circumvented, see logic in getPatchItemWithSchema()
  def getStrippedPartitionKeyPath(partitionKeyDefinition: PartitionKeyDefinition): String = {
```

Comment:
can remove the TODO then

and also since the strippedPartitionKeyPath is not being used in the getPatchItemWithFullSchemaSubpartitions, I think it might be better to remove the partitionKeyPath parameter from the method